CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v0.29.1
+
+*January 24, 2019*
+
+Special thanks to external contributors on this release:
+@infinytum, @gauthamzz
+
+This release contains two important fixes: one for p2p layer where we sometimes
+were not closing connections and one for consensus layer where consensus with
+no empty blocks (`create_empty_blocks = false`) could halt.
+
+Friendly reminder, we have a [bug bounty
+program](https://hackerone.com/tendermint).
+
+### BREAKING CHANGES:
```

Comment:
forgot to remove empty sections, but now can't push because I don't have write access to release branches

Reply:
heh - I applied the same permissions to `release/*` as we have on develop. so you'd need to make a PR, have it approved, and squash merge

Reply:
I'll do it - thanks!